Embedded/setup.in.py

```diff
@@ -7,12 +7,22 @@
 from distutils.core import setup, Extension
 
 import os
+import sys
 import numpy as np
 import pyarrow as pa
 
 root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# conda-forge packages omniscidbe and pyomniscidbe are built
+# separately. OMNISCI_ROOT_PATH is defined by the omniscidbe activate
+# script that determines the location of libDBEngine.so and is
+# required in linking Python extension module omniscidbe.
+omnisci_root_path = os.environ.get('OMNISCI_ROOT_PATH',
```

Comment:
Sure, it is defined in omniscidbe activate script:

https://github.com/conda-forge/omniscidb-feedstock/pull/39/files#diff-950dd1a30450006d4e2daa139eb257c38e5ffa2cc5f0dabbe9b12dbf485f0d00R213

Reply:
Thus, the definition of this variable must also be added to omnisci/scripts?

Reply:
No, because `omniscidb/scripts/conda` scripts use the same build directory for all components, hence `build/Embedded/libDBEngine.so` exists when running `setup.py` 

The above does not apply to omniscidb-feedstock where `omniscidbe` and `pyomniscidbe` packages are built separately.

Reply:
But in setup.py, without any conditions, you take a non-existent variable, build a path relative to it and add it to library_dirs. Are you sure there are no issues?

Reply:
Fixed: `OMNISCI_ROOT_PATH` is used only when defined.